.github/workflows/slack-message-broker.yml

```diff
@@ -0,0 +1,49 @@
+# This workflow can be used to notify Slack channels when workflow_run events occur.
+# We currently notify the plutus-core-dev and plutus-tools-dev channels whenever
+# the script-evaluation-test or benchmark workflows run.
+# TODO this must be turned into a standalone GitHub action so that it can be used 
+# for arbitrary channels and workflows. 
+name: Slack Message Broker
+on:
+  workflow_run:
+
+jobs:
+  slack-broker:
+    runs-on: [ubuntu-latest]
+
+    steps:
+
+      - name: Prepare Slack Message
+        uses: actions/github-script@v6
+        id: prepare-slack-message
+        if: contains(fromJSON('["script-evaluation-test", "benchmark"]'), github.event.workflow_job.name)
```

Comment:
I think I'd mildly prefer it if the direction of causality went the other way around? Can we explicitly trigger this workflow at the end of the other workflows?

Reply:
The line `if: contains(fromJSON('["script-evaluation-test", "benchmark"]'), github.event.workflow_job.name)` is invalid. I've changed this in the latest commit